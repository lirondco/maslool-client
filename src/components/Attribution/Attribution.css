.attribution {
    position: fixed;
    left: 0;
    bottom: 0;
    width: 100%;
    justify-content: center;
    flex-wrap: wrap;
    display: flex;
    flex-direction: row;
    padding-top: 5px;
}

.attribution, .attribution p, .attribution .separate_link {
    background-color: #374355;
    color: #f3f4f5;
    font-size: xx-small;
    padding-left: 2%;
    padding-right: 2%
}

.attribution a {
    color: #f9d4bf !important;
    font-weight: bold !important;
    background-color: #374355;
}

