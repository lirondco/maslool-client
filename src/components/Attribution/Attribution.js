import React from 'react'
import './Attribution.css'

export default function Attribution () {
    return (
        <footer className='attribution'>
            <p>Created by Liron de Castro for her final capstone at <a href='https://www.thinkful.com/'>Thinkful</a>.</p>
            <a href='https://www.linkedin.com/in/liron-de-castro/' className='separate_link'>LinkedIn</a>
            <a href='https://github.com/lirondco' className='separate_link'>GitHub</a>
            <p>Tel Aviv, Israel x San Diego, CA</p>
            <p lang='zh-Hant'>人民有顛覆暴政的自由</p>
        </footer>
    )
}