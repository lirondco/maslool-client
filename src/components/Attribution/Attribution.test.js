import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Attribution from './Attribution'

describe(`Attribution component`, () => {
    it('renders the footer', () => {
        const wrapper = shallow(<Attribution />)
        expect(toJson(wrapper)).toMatchSnapshot()
    })
})