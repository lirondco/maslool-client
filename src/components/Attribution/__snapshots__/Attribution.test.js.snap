// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Attribution component renders the footer 1`] = `
<footer
  className="attribution"
>
  <p>
    Created by Liron de Castro for her final capstone at 
    <a
      href="https://www.thinkful.com/"
    >
      Thinkful
    </a>
    .
  </p>
  <a
    className="separate_link"
    href="https://www.linkedin.com/in/liron-de-castro/"
  >
    LinkedIn
  </a>
  <a
    className="separate_link"
    href="https://github.com/lirondco"
  >
    GitHub
  </a>
  <p>
    Tel Aviv, Israel x San Diego, CA
  </p>
  <p
    lang="zh-Hant"
  >
    人民有顛覆暴政的自由
  </p>
</footer>
`;
